pkg/api/meta/meta.go

```diff
@@ -506,8 +548,20 @@ func (a genericAccessor) SetAnnotations(annotations map[string]string) {
 	*a.annotations = annotations
 }
 
-func (a genericAccessor) GetOwnerReferences() []unversioned.ObjectReference {
-	return a.ownerReferences
+func (a genericAccessor) GetOwnerReferences() []OwnerRefereceAccessor {
```

Comment:
This interface has the advantage that the standard object meta can implement it without too much trouble, just needs a helper type for the accessor.